language/move-prover/tests/testsuite.rs

```diff
@@ -251,14 +259,28 @@ fn collect_enabled_tests(reqs: &mut Vec<Requirements>, group: &str, feature: &Fe
     let mut p = PathBuf::new();
     p.push(path);
     for entry in WalkDir::new(p.clone())
-        .follow_links(true)
+        .follow_links(false)
         .min_depth(1)
         .into_iter()
         .flatten()
     {
         if !entry.file_name().to_string_lossy().ends_with(".move") {
             continue;
         }
+
+        // TODO: this is to handle the awkwardness in the experimental folder in diem-framework
```

Comment:
Packages will be used soon, it would be good to land this soon as I'm going to refactor this.

Reply:
Yes, package should be the solution.

The only thing that blocks it from landing is that the new `DiemAccount.move` in the experimental dir does not pass the verification... I could temporarily turn it off and the refactoring team can debug it later.